core/runtime/src/main/java/io/quarkus/runtime/util/ContainerRuntimeUtil.java

```diff
@@ -225,7 +222,7 @@ public enum ContainerRuntime {
         private final boolean rootless;
 
         ContainerRuntime(String executableName, boolean rootless) {
-            this.executableName = executableName + (OS.current() == OS.WINDOWS ? ".exe" : "");
```

Comment:
This is superfluous and thus confusing. Regardless of podman reporting `podman` or `podman.exe` it is always legal to call `podman`. Similar with e.g. `mspaint` vs. `mspaint.exe`,